Sources/Prototyper/Feedback/EditScreenshotView.swift

```diff
@@ -35,101 +34,131 @@ struct EditScreenshotView: View {
-            .sheet(isPresented: $colorPickerShown) {
-                NavigationView {
-                    ColorPickerView(color: self.$color, colorPickerShown: self.$colorPickerShown)
-                        .navigationBarItems(leading: self.cancelButton)
-                }
-            }
+        }.navigationBarTitle("Markup")
+        .navigationBarItems(leading: cancelButton, trailing: saveButton)
+        .sheet(isPresented: $colorPicker) {
+            ColorPicker("Pick Color", selection: $color)
+                .padding()
+        }
     }
     
-    /// The color picker, undo and clear actions for the Markup View.
+    ///View for all drawing related actions
     var actions: some View {
         HStack(alignment: .center, spacing: 30) {
-            Image(systemName: "eyedropper.halffull")
+            Image(systemName: pencilImage)
                 .imageScale(.large)
                 .onTapGesture {
-                    self.colorPickerShown.toggle()
+                    isDraw.toggle()
                 }
-            Image(systemName: "arrow.uturn.left")
+            Image(systemName: inkTypeImage)
                 .imageScale(.large)
                 .onTapGesture {
-                    if self.allDrawings.isEmpty {
-                        self.allDrawings.removeLast()
-                    }
+                    changeInkType()
                 }
-            Image(systemName: "xmark")
+            Image(systemName: "pencil.circle.fill")
                 .imageScale(.large)
                 .onTapGesture {
-                    self.allDrawings = [Drawing]()
+                    colorPicker.toggle()
                 }
         }.frame(height: 32)
     }
     
-    /// This view records the strokes and gestures of the user on the current View.
-    var dragGesture: some Gesture {
-        DragGesture(minimumDistance: 1, coordinateSpace: .global)
-            .onChanged { value in
-                if self.rect.contains(value.location) {
-                    if self.currentDrawing == nil {
-                        self.currentDrawing = Drawing(color: self.color)
-                    }
-                    self.currentDrawing?.points.append(value.location)
-                }
-            }
-            .onEnded { _ in
-                self.currentDrawing.map { self.allDrawings.append($0) }
-                self.currentDrawing = nil
-            }
-    }
-    
     /// The cancel button displayed on the left top corner of the View.
     private var cancelButton: some View {
-        Button(action: cancel) {
+        Button(action: cancle) {
             Text("Cancel")
         }
     }
     
     /// The save button displayed on the top right corner of the View.
     private var saveButton: some View {
-        Button(action: save) {
+        Button(action: saveDrawing) {
             Text("Save").bold()
         }
     }
     
+    ///Adds the drawings to the screenshot
+    func saveDrawing() {
```

Comment:
Maybe the functionality of this method as well as the `combineImage` method would be better situated in the `DrawingBoard` view?